src/index.ts

```diff
@@ -1,7 +1,10 @@
 import jwt_decode from 'jwt-decode';
 import { Claims, defaultOptions, Options } from './types';
 import { Configuration, DefaultApi, Target } from './openapi';
-import { VERSION } from './utils';
+import { log, VERSION } from './utils';
+import { PollingProcessor } from './polling';
+import { StreamProcessor } from './streaming';
+import * as events from 'events';
 
 export class CfClient {
```

Comment:
Not sure I'd consider this in the scope of this ticket, but since we no longer use _"Continuous Features"_, should this be `FFClient`?